sass/directives/_carousels.scss

```diff
@@ -162,15 +157,16 @@
 }
 
 @mixin testimonial__carousel__logo {
```

Comment:
if you name your logo class `.[whatever your container class is]__img-wrap` then the following styles are automatically applied via `@media-item` and you could remove lines 163-5. Or if that doesn't work for some reason at least use the flex mixins

```
    @include flex-grow(0);
    @include flex-shrink(0);
    @include flex-basis(auto);
    width: $image-wrap-width;
    margin-right: $base-spacing;

    @media only screen and (max-width: $breakpoint) {
      width: 100%;
      margin-right: 0;
      margin-bottom: $base-margin;
    }
```